Models/AssetComparer.cs

```diff
@@ -59,14 +64,31 @@ private void UpdateAssetStates(IEnumerable<T> intersectingAssets)
         {
             foreach (T currentAsset in currentAssets)
             {
-                if (!intersectingAssets.Any(asset => asset.AssetId == currentAsset.AssetId))
+                T foundIntersectingAsset =
```

Comment:
Mangler indrykning

Reply:
Should be OK in the code. Could be a Github thing

Reply:
Never mind my bad